src/components/settings/SettingsSSOTab/tests/NewSSOConfigForm.test.jsx

```diff
@@ -103,7 +111,11 @@ describe('SAML Config Tab', () => {
     render(
       <Provider store={store}>
         <SSOConfigContext.Provider value={contextValue}>
-          <Provider store={store}><NewSSOConfigForm enterpriseId={enterpriseId} /></Provider>
+          <NewSSOConfigForm
+            enterpriseId={enterpriseId}
+            setIsStepperOpen={jest.fn()}
```

Comment:
nit: why aren't we using `setIsStepperOpen` as defined above?

Reply:
uhhhh I think the mocked func came later when I was cleaning stuff up. I can backfill it